crypten/__init__.py

```diff
@@ -330,7 +330,10 @@ def load(f, load_closure=torch.load, **kwargs):
         to the closure.
     """
     # TODO: Add support for loading from correct device (kwarg: map_location=device)
-    obj = load_closure(f, **kwargs)
+    if load_closure == torch.load:
```

Comment:
Q: I am not sure this is the best approach to fix this.

Reply:
It's not pretty but it seems fine to me.